src/Pages/H2P.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import '../App.css';
+import '../Styles/H2P.css';
+import Navbar from '../Components/LandingNavbar/Navbar'
+
+function H2P() {
+  return (
+    <>
+    <Navbar />
+      <div className='how2playpage'>
```

Comment:
dashes for spaces instead of botched camel case.
that "2" does not belong there